rpm-build/SPECS/securedrop-workstation-dom0-config.spec

```diff
@@ -168,6 +174,22 @@ find /srv/salt -maxdepth 1 -type f -iname '*.top' \
 # mkdir -p /tmp/sdw-migrations
 # touch /tmp/sdw-migrations/whonix-17-update
 
+# Adjust icon size (usability/accessibility)
+sudo -u $(sudo groupmems -l -g qubes) /opt/securedrop/update-xfce-settings adjust-icon-size ||:
+sudo -u $(sudo groupmems -l -g qubes) /opt/securedrop/update-xfce-settings disable-unsafe-power-management ||:
+
```

Comment:
(I'm not sure if I really like this, and I still have to test it. This would enable the power settings as soon as the rpm is installed, but then on dev machines the change wouldn't be reverted til the reboot after `sdw-admin --apply`.)